---
layout: gridlay
title: Research
subtitle: Corces Lab Research
---

<style>
	.right {
    float: right;
    padding: 10px 10px 10px 10px;
	}
	.left {
    float: left;
    padding: 10px 10px 10px 10px;
	}
</style>

<div align="center">
	<h1>
		<strong>Research in the Corces Lab</strong>
	</h1>
</div>
<hr>
<!-- The paddingtop and margin-top edits allow anchors to link properly. -->
<div class="container">
  <div class="jumbotron jumbotron-correct">
      <p>
        In the Corces Lab, we aim to understand one fundamental question: <b> why do some individuals develop neurodegenerative disease while others do not?</b> The answer to this question, broadly speaking, boils down to “nature and nurture”, or genetics and environment, and we approach both of these angles simultaneously by studying the noncoding genome and the epigenetic control of disease-related cellular phenotypes. While much of our work in the lab focuses on the two most common neurodegenerative diseases, Alzheimer’s disease (AD) and Parkinson’s disease (PD), we collaborate with other labs to study disease contexts ranging from traumatic brain injury to glaucoma.
      </p>
  </div>
</div>

<hr>

<div id="Main focus #1" class="col-sm-12">
	<h4>Main focus #1: Noncoding genetics in neurodegenerative disease</h4>
	<div class="square" style="text-align: justify;">
		<img src="/img/research/ADHeritability.jpg" alt="Heritability of AD" class="right" width="300">
		<p>
			Despite decades of work correlating genetic variation to disease, the vast majority of disease predisposition remains unexplained. AD and other complex diseases are estimated to be 40-60% heritable. Yet our current best methods explain less than half of that predisposition. In practical terms, this means that for many patients, no therapeutics may be available or in the pipeline to counteract their disease. Why are we still missing so many of the genetic determinants of complex diseases? Though cliché, the streetlight analogy is highly applicable here – we tend to search for disease drivers among variants in the coding part of the genome, because it is easy to nominate the genes these variants disrupt. But coding regions represent approximately 2% of the human genome and the vast majority (> 95%) of variants map to noncoding regions of the genome!
		</p>
		<p>
			The genetics of complex disease has largely been studied through the use of genome-wide association studies (GWASs). These studies have identified thousands of genetic variants, coding or noncoding, that correlate with an increased risk of developing AD or PD. However, few actionable therapeutic interventions have ensued. This is largely because GWASs are not specific enough: they home in on genomic loci associated with disease, but they do not pinpoint which specific variants are functional, nor do they identify the genes affected by variants that do not fall within a coding region. 
		</p>
		<p>
			Noncoding variants primarily exert their effects by altering the sequence-specific binding of transcription factors (TFs), which ultimately changes where and when genes are expressed. A primary goal of the lab is to define the functional noncoding variants impacting predisposition to AD and PD. To this end, we integrate large-scale multi-omic  profiling of patient tissues, state-of-the-art machine learning models, and high-throughput screens in iPSC-derived models of key brain cell types. By broadening our scope to the vast and underexplored noncoding genome, we expect to identify new genetic determinants of AD and PD pathogenesis and nominate new putative targets for therapeutic intervention.
		</p>
	</div>
</div>

<hr>
<div id = "Selective Vulnerability" class="row" style="padding-top: 60px; margin-top: -60px;">
    <div class="col-sm-8" style="text-align: justify">
    	<h5>Selective Vulnerability in AD</h5>
    	<p>
    		In addition to the cognitive resilience to pathology mentioned above, AD is also characterized by “selective vulnerability” – the disease only affects particular neurons and exhibits consistent differential effects on different brain regions. Moreover, this selective vulnerability exists across multiple diverse neurodegenerative diseases, though the neuronal types affected by the pathology differ across diseases. In most cases, the cells that accumulate the pathology-associated proteins are the cells that are most affected. However, many examples exist of neuronal cell types that accumulate pathology but remain relatively unaffected. Some studies indicate that this is a cell-autonomous property of the cells, implicated specific gene expression patterns in the selectivity of pathologic vulnerability. We are using single-cell approaches to characterize cell type- and brain region-specific differences to identify putative mechanisms underlying this selective vulnerability. 
    	</p>
    </div>
    <div class="col-sm-4">
    	<img src="/img/research/SelectiveVulnerability.jpg" alt="Selective Vulnerability in AD">
    </div>
</div>
<hr>
<div id = "Inherited Variaion in AD" class="row" style="padding-top: 60px; margin-top: -60px;">
	<div class="col-sm-4">
    	<img src="/img/research/GWAS.jpg" alt="Inherited Variaion in AD">
    </div>
    <div class="col-sm-8" style="text-align: justify">
    	<h5>The role of inherited genetic variation in AD</h5>
    	<p>
    		Through decades of research, genome-wide association studies have identified heritable mutations that lead to an increased risk of developing AD. The vast majority (>95%) of these mutations occur in noncoding regions of the genome, making their functions difficult to predict. As such, many of these mutations remain under-characterized and their contribution to AD pathogenesis remains unclear. Moreover, it has become increasingly clear that an individual’s lifetime risk of developing AD is not merely governed by genetics. In addition, the epigenome, the complement of all of the chemical and physical modifications imposed on DNA that do not change the underlying sequence, is also thought to play a crucial role. We aim to define the epigenetic and genetic components of AD through profiling of the open chromatin landscapes and three-dimensional chromatin interactions in brain regions and primary cell types of patients with and without AD. We have integrated machine learning and single-cell epigenomics to predict functionality of many of these polymorphisms. Layering on single-cell transcriptomics, massively parallel reporter assays, and CRISPR-based perturbation of regulatory elements, we hope to uncover the mechanisms behind AD-associated genetic variation with the ultimate goal of identifying novel aspects of AD pathogenesis and nominating putative avenues for therapeutic intervention.
    	</p>
    </div>
</div>

<hr>

<div id="Parkinson’s Disease" class="col-sm-12">
		<h3>Parkinson’s Disease</h3>
		<p>
			The Corces Lab is also broadly interested in other neurodegenerative diseases including Parkinson’s disease (PD). PD is a progressive long-term neurodegenerative disorder that is classically associated with effects on the motor system (tremors, muscle rigidity, bradykinesia etc.). The primary cellular cause of these symptoms is the loss of dopamine-secreting neurons in the substantia nigra. Similar to the pathologic accumulations in AD, PD is characterized by the accumulation of alpha-synuclein protein in Lewy bodies. Many of the same questions and techniques that we apply to AD are also being applied to PD. We are particularly interested in the inherited genetic contributions to PD as our understanding of this remains much less mature.
		</p>
</div>